polyfills/Element/prototype/matches/config.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"browsers": {
 		"ie": "6 - *",
+		"ie_mob": "10 - *",
```

Comment:
To make multiple PRs, you would need to create a separate branch for each PR.

@kentcdodds has made a great video tutorial on how to contribute to open source projects on Github. -- https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github